common/common.go

```diff
@@ -45,19 +46,27 @@ import (
 	"github.com/fnproject/fn_go/modelsv2"
 	"github.com/spf13/viper"
 	"github.com/urfave/cli"
-	"gopkg.in/yaml.v2"
 )
 
 // Global docker variables.
 const (
-	FunctionsDockerImage     = "fnproject/fnserver"
+	//FunctionsDockerImage     = "fnproject/fnserver"
+	FunctionsDockerImage     = "greendragons/testfnproject"
```

Comment:
Since we have not yet released changes in fnserver so it is currently pointing to my custom fn image which has the relevant changes. Once we release fnserver image then I'll revert it back. If I do it now then tests would start failing.